src/js/components/search/table/cells/ResultsTableHeaderCell.jsx

```diff
@@ -1,112 +1,104 @@
 /**
-  * ResultsTableHeaderCell.jsx
-  * Created by Kevin Li 12/1/16
-  **/
+ * ResultsTableHeaderCell.jsx
+ * Created by Kevin Li 12/8/17
+ */
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import * as Icons from 'components/sharedComponents/icons/Icons';
+import { ArrowUp, ArrowDown } from 'components/sharedComponents/icons/Icons';
 
 const propTypes = {
-    label: PropTypes.string,
-    column: PropTypes.string,
+    isLast: PropTypes.bool,
+    isActive: PropTypes.bool,
+    title: PropTypes.string,
     defaultDirection: PropTypes.string,
-    order: PropTypes.object,
-    setSearchOrder: PropTypes.func,
-    isLastColumn: PropTypes.bool
+    currentSort: PropTypes.object,
+    updateSort: PropTypes.func
 };
 
-export default class ResultsTableHeaderCell extends React.Component {
-    constructor(props) {
-        super(props);
+const TableHeaderCell = (props) => {
+    const clickedSort = (e) => {
+        props.updateSort(props.title, e.target.value);
+    };
 
-        this.clickedHeader = this.clickedHeader.bind(this);
-        this.forceDirection = this.forceDirection.bind(this);
-    }
-
-    clickedHeader() {
-        // check if this is the field that is currently being used to sort
-        if (this.props.column === this.props.order.field) {
-            // it's the same field, just toggle the direction
-            let direction = 'asc';
-            if (this.props.order.direction === 'asc') {
-                direction = 'desc';
-            }
+    const clickedDefault = () => {
+        // if (props.isActive) {
```

Comment:
Do we need this commented-out code?

Reply:
Yes, this is the old code that enables ascending sort, for when the backend is capable of doing it again.